### PR TITLE
Remove ESA version from ember inspector

### PR DESCRIPTION
**Pull request by brunoocasali**
Closes https://github.com/simplabs/ember-simple-auth/issues/2303
Closes https://github.com/simplabs/ember-simple-auth/issues/2309

### CODE REVIEW

File: packages/ember-simple-auth/index.js [hunk 1]
```diff
@@ -3,37 +3,26 @@
 /* eslint-env node */
 /* eslint-disable no-var, object-shorthand, prefer-template */
 
-var writeFile = require('broccoli-file-creator');
```
Comment (josemarluedke) [inline packages/ember-simple-auth/index.js:6]: Are these deps not used anywhere else? If not, these can be removed.

Comment (josemarluedke) [reply packages/ember-simple-auth/index.js:6]: `broccoli-file-creator`, `broccoli-merge-trees`

Comment (brunoocasali) [reply packages/ember-simple-auth/index.js:6]: Good catch, solved in https://github.com/simplabs/ember-simple-auth/pull/2321/commits/5e6377b347ffcee856ded5164d178dd82e636475 thank you!

**Review by josemarluedke — APPROVED** [review]

**Review by BobrImperator — APPROVED** [review]
Looks good to me

**Review by marcoow — APPROVED** [review]

**Comment by BobrImperator** [comment]
It seems like we have some linting issues on this PR, could you please fix them @brunoocasali?

**Comment by brunoocasali** [comment]
@BobrImperator could you please accept the workflow execution?

**Pull request merged**